lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -152,10 +194,29 @@ class FlutterConfiguration {
   /// ```
   /// flutter run -d chrome --profile --dart-define=FLUTTER_WEB_DEBUG_SHOW_SEMANTICS=true
   /// ```
-  bool get debugShowSemanticsNodes => _js?.debugShowSemanticsNodes ?? _defaultDebugShowSemanticsNodes;
+  bool get debugShowSemanticsNodes => getConfigValue<bool>(
+    (JsFlutterConfiguration c) => c.debugShowSemanticsNodes,
+    _defaultDebugShowSemanticsNodes);
   static const bool _defaultDebugShowSemanticsNodes = bool.fromEnvironment(
     'FLUTTER_WEB_DEBUG_SHOW_SEMANTICS',
   );
+
+  /// Returns the [targetElement] in which the Flutter Application is supposed
```

Comment:
`targetElement` should be **`hostElement`** everywhere, as agreed upon in the design doc.